load/UpdateDBPLicensorTables.py

```diff
@@ -52,18 +52,13 @@ def processFileset(self, primaryFilesetId, derivedHashId):
 
 		for (copyright, copyrightDate, copyrightDescription) in copyrights:
 			hashIdExisting = self.db.selectScalar("\
-				SELECT bible_fileset_copyrights.copyright,\
-					bible_fileset_copyrights.copyright_date,\
-					bible_fileset_copyrights.copyright_description,\
-					bible_fileset_copyrights.open_access\
+				SELECT bible_fileset_copyrights.copyright\
```

Comment:
this seems unrelated. 
and it removes copyright_date. 

I think we need to be more focused, so that the PR description correctly represents the code being changed.

Does copyright_date need to be added back in?

Reply:
@bradflood, I made this change because, while investigating the issue (task-2371), I noticed that the ETL process was always creating a SQL statement to update the copyright of a derived fileset using the same existing information. As a result, I removed this no longer necessary line: [link](https://github.com/faithcomesbyhearing/dbp-etl/pull/170/files#diff-5bd192f19936aa1de297001dae83cd0ca789f285243182525ab307410295613dL65).

Regarding the query, it won't remove the copyright_date. This query is used to check whether the derived fileset has copyright content. The copyright_date value is retrieved from the primary fileset in the following line: [link](https://github.com/faithcomesbyhearing/dbp-etl/blob/75bf8788e8a6c41068d341f69588df6c0fbdc11f/load/UpdateDBPLicensorTables.py#L51).